_includes/v20.1/orchestration/kubernetes-upgrade-cluster.md

```diff
@@ -68,21 +67,45 @@ Kubernetes knows how to carry out a safe rolling upgrade process of the Cockroac
 
     {% endif %}
 
-    2. Set the `cluster.preserve_downgrade_option` [cluster setting](cluster-settings.html):
+    1. Set the `cluster.preserve_downgrade_option` [cluster setting](cluster-settings.html):
 
         {% include copy-clipboard.html %}
         ~~~ sql
         > SET CLUSTER SETTING cluster.preserve_downgrade_option = '19.2';
         ~~~
 
-    3. Exit the SQL shell and delete the temporary pod:
+    1. Exit the SQL shell and delete the temporary pod:
 
         {% include copy-clipboard.html %}
         ~~~ sql
         > \q
         ~~~
 
-2. Kick off the upgrade process by changing the desired Docker image:
+1. Add a [partition](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#staging-an-update) to the `updateStrategy` defined in the StatefulSet. Only the pods numbered greater than or equal to the partition value will be updated. For a cluster with 3 pods (e.g., `cockroachdb-0`, `cockroachdb-1`, `cockroachdb-2`) the partition value should be 2:
+
+    <section class="filter-content" markdown="1" data-scope="manual">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ kubectl patch statefulset cockroachdb \
+    -p='{"spec":{"updateStrategy":{"type":"RollingUpdate","rollingUpdate":{"partition":2}}}}'
+    ~~~
+
+    ~~~
+    statefulset.apps/cockroachdb patched
+    ~~~
+    </section>
+
+    <section class="filter-content" markdown="1" data-scope="helm">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ helm upgrade \
+    my-release \
+    cockroachdb/cockroachdb \
+    --set statefulset.updateStrategy.rollingUpdate.partition=2 \
+    ~~~
+    </section>
+
+1. Kick off the upgrade process by changing the desired Docker image:
```

Comment:
Suggestion: "desired Docker image" > "Docker image used in the CockroachDB StatefulSet "